README.md

```diff
@@ -176,7 +176,7 @@ Please see the [type specification](https://github.com/node-saml/node-saml/blob/
 
 ##### **InResponseTo Validation**
 
-- `validateInResponseTo`: if truthy, then InResponseTo will be validated from incoming SAML responses
+- `validateInResponseTo`: if set to `always` or `ifPresent`, then InResponseTo will be validated from incoming SAML responses. The default setting is `never`, so this setting needs to be set if you want to validate incoming SAML responses.
```

Comment:
Let's change the default to "always" for `master` and have another PR that uses this verbiage against the 4.x branch.

Reply:
There is no 4.x branch in this repo